### PR TITLE
DOC init docs file

### PR DESCRIPTION
**Pull request by ganncamp**

### CODE REVIEW

File: sonar-docs/analysis/scan/sonarscanner-for-azure-devops.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+<!-- sonarcloud -->
+Microsoft has published a [dedicated lab](https://aka.ms/sonarcloudlab) describing how to integrate Azure DevOps Pipelines and SonarCloud. The lab includes setting up a Branch Policy in Azure DevOps to block a Pull Request from being submitted if the changed code does not meet the quality bar.
+<!-- /sonarcloud -->
+
+## Compatibility
+Version <!-- sonarqube -->4.x<!-- /sonarqube --><!-- sonarcloud -->1.x<!-- /sonarcloud --> is compatible with:
+* TFS 2017 Update 2+
+* TFS 2018
+* Azure DevOps Server 2019
+* Azure DevOps
```
Comment (duncanp-sonar) [inline sonar-docs/analysis/scan/sonarscanner-for-azure-devops.md]: The SonarCloud extension can only be installed on Azure DevOps.

File: sonar-docs/analysis/scan/sonarscanner-for-azure-devops.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+<!-- sonarqube -->
+The SonarQube Extension embeds its own version of the [SonarScanner for MSBuild](/analysis/scan/sonarscanner-for-msbuild/).
+<!-- /sonarqube -->
```
Comment (duncanp-sonar) [inline sonar-docs/analysis/scan/sonarscanner-for-azure-devops.md]: This is true for the SonarCloud extension too.

File: sonar-docs/analysis/scan/sonarscanner-for-azure-devops.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+
+## Installation
+1. Install the extension [from the marketplace](https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarqube).
```
Comment (duncanp-sonar) [inline sonar-docs/analysis/scan/sonarscanner-for-azure-devops.md]: Needs a separate link for SonarCloud:
https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarcloud

File: sonar-docs/analysis/scan/sonarscanner-for-azure-devops.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+## Configure
+The first thing to do is to declare <!-- sonarqube -->your SonarQube server<!-- /sonarqube --><!-- sonarcloud -->SonarCloud<!-- /sonarcloud --> as a service endpoint in your Azure DevOps project settings. 
+
+1. Open the Connections page in your Azure DevOps project: **Project Settings > Pipelines > Service Connections**.
+1. Click on **New service connection** and choose **{instance}**.
+<!-- sonarqube -->
+1. Specify a **Connection name**, the **Server URL** of your SonarQube Server (including the port if required) and the [Authentication Token](/user-guide/user-token/) to use.
+<!-- /sonarqube -->
+<!-- sonarcloud -->
+1. Specify a **Connection name** and **SonarCloud token**. There is a link in the dialog that will take you to the account security page on SonarCloud where you can create a new token if necessary. You can also button that lets you verify that connection is correctly configured.
```
Comment (duncanp-sonar) [inline sonar-docs/analysis/scan/sonarscanner-for-azure-devops.md]: > You can also button...

File: sonar-docs/analysis/scan/sonarscanner-for-azure-devops.md (hunk 1 above)
Comment (ganncamp) [reply sonar-docs/analysis/scan/sonarscanner-for-azure-devops.md]: At this point there's no point in having a compatibility section for SC, so I've scoped the section to just SQ & dropped ADO Services.

**Pull request merged**